libs/trustchain/src/mockSdk.ts

```diff
@@ -60,27 +57,27 @@ export class MockSDK implements TrustchainSDK {
     });
   }
 
-  async authWithDevice(transport: Transport): Promise<JWT> {
-    void transport;
-    return Promise.resolve(mockedSeedIdAccessToken);
-  }
-
-  async auth(trustchain: Trustchain, memberCredentials: MemberCredentials): Promise<JWT> {
+  withAuth<T>(
+    trustchain: Trustchain,
+    memberCredentials: MemberCredentials,
+    f: (jwt: JWT) => Promise<T>,
```

Comment:
could you avoid using signe char to name function/const ? 🫣 I know it's for test purpose but it more easy to read !